### ISSUE TITLE
Make MAXO normalisation release

### ISSUE THREAD

**Issue by matentzn** [monarch-initiative/MAxO#457]
After the many changes to the pipeline (maybe after fixing https://github.com/monarch-initiative/MAxO/issues/443) we should make an intermediate release with the new ODK (before adding all these new terms Peter wants). I can do it if you want after #443 is finished.

@LCCarmody let me know if you need help!

**Comment by LCCarmody** [comment]
I do need your help. I attempted to do a release (published branch feb2023release).  After running prepare_release, all of the     obsolete and replaced template terms turned up with errors for lack of definitions. Also, I'm getting an error for the terms that are replaced with as well.  Ex. MAXO:0000061 is throwing an error for missing label, but it has a label. The issue seems to be MAXO:0000204 which was replaced by MAXO:0000061. I can create a PR and you fix that? And #443, while important, changes the definitions of the terms--and is not technically a bug. Needs re-writing the text definition should fix it. JUst thinking I should obsolete those IDs since changing text def.
